aks/examples/cnpack/terraform.tfvars

```diff
@@ -1,13 +1,16 @@
 # SPDX-FileCopyrightText: Copyright (c) 2022-2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-// Cluster Variables
-# cluster_name = "cnpack"
-# location = "West US 2"
-# admin_group_object_ids = []
+# Sample tfvars file. Uncomment out values to use
+# Do not commit this file to Git with sensitive values
 
-// Fluentbit/Azure Log Configuration Variables
-# fluentbit-workspace-name = "fluentbit-test"
-
-// Prometheus/Azure Monitor Configuration Variables
-# prometheus-name = "cnpack-prometheus"
+# admin_group_object_ids                = ""
+# az_monitor-user-managed-id            = "tf-holoscan-identity"
+# azure_log_analytics_retention_in_days = 30
+# azure_log_analytics_sku               = "PerGB2018"
+# cluster_name                          = ""
+# fluentbit-workspace-name              = ""
+# fluentbit_enabled                     = true
+# location                              = ""
+# prometheus-name                       = ""
```

Comment:
Any chance we can fix the naming convention in this MR? Most of them have _ as separator, but this one and " az_monitor-user-managed-id" have "-" as separator

Reply:
Done -- fixed both